examples/plugin/load/main.cc

```diff
@@ -24,7 +24,11 @@ int main(int argc, char *argv[])
   std::ifstream config_in{argv[2]};
   if (!config_in.good())
   {
-    std::cerr << "Failed to open config file: " << std::strerror(errno) << "\n";
+    char err_msg[256] = {
+        '\0',
+    };
+    strerror_s(err_msg, sizeof(err_msg), errno);
+    std::cerr << "Failed to open config file: " << err_msg << "\n";
```

Comment:
```suggestion
    std::perror("Failed to open config file");
```

This should be safe across platforms.